code/modules/media/jukebox.dm

```diff
@@ -405,20 +427,26 @@ var/global/loopModeNames=list(
 		attack_hand(user)
 
 /obj/machinery/media/jukebox/emag(mob/user)
-	current_song = 0
 	if(!emagged)
-		playlist_id = "emagged"
-		last_reload=world.time
-		playlist=null
-		loop_mode = JUKEMODE_SHUFFLE
-		emagged = 1
-		playing = 1
 		user.visible_message("<span class='warning'>[user.name] slides something into the [src.name]'s card-reader.</span>","<span class='warning'>You short out the [src.name].</span>")
-		update_icon()
-		update_music()
-		return 1
+		wires.CutWireIndex(JUKE_CONFIG)
 	return
 
+/obj/machinery/media/jukebox/proc/short()
```

Comment:
This may be a semantic argument, but my understanding is that the "point" of emagging a juke is not that no one else can do it, nor to damage the jukebox so that it can't be used in the previous way. The point is to get at the interesting, alternate song list.

This is similar to a door -- you're trying to get to an objective. It's easier with an emag, but not impossible without one. The goal, however, isn't the hackjob itself: it's what's on the other sideof that door.